iroh/src/endpoint.rs

```diff
@@ -1468,34 +1468,35 @@ mod tests {
 
         info!(time = ?test_start.elapsed(), "test setup done");
 
-        let barrier = Arc::new(tokio::sync::Barrier::new(2));
-
         // The server accepts the connections of the clients sequentially.
-        let s_b = barrier.clone();
         let server = tokio::spawn(
             async move {
                 let eps = ep.bound_sockets();
 
                 info!(me = %ep.id().fmt_short(), eps = ?eps, "server listening on");
                 for i in 0..n_clients {
-                    let round_start = Instant::now();
-                    info!("[server] round {i}");
-                    let incoming = ep.accept().await.anyerr()?;
-                    let conn = incoming.await.anyerr()?;
-                    let endpoint_id = conn.remote_id();
-                    info!(%i, peer = %endpoint_id.fmt_short(), "accepted connection");
-                    let (mut send, mut recv) = conn.accept_bi().await.anyerr()?;
-                    let mut buf = vec![0u8; chunk_size];
-                    for _i in 0..n_chunks_per_client {
-                        recv.read_exact(&mut buf).await.anyerr()?;
-                        send.write_all(&buf).await.anyerr()?;
-                    }
-                    info!(%i, peer = %endpoint_id.fmt_short(), "finishing");
-                    send.finish().anyerr()?;
-                    conn.closed().await; // we're the last to send data, so we wait for the other side to close
-                    info!(%i, peer = %endpoint_id.fmt_short(), "finished");
-                    info!("[server] round {i} done in {:?}", round_start.elapsed());
-                    s_b.wait().await;
+                    tokio::time::timeout(Duration::from_secs(4), async {
```

Comment:
Can we use a constant for `Duration::from_secs(4)` with here and below as well?
Ideally with a decent comment about how RTT should usually be sub-100ms but shutdown can take 3s due to connection draining and being unsure about datagrams still flying around on holepunching attempts.